src/actions/onboardingActions.js

```diff
@@ -547,7 +551,7 @@ export const checkUsernameAvailabilityAction = (username: string) => {
       'checkUsernameAvailabilityAction: done, dispatching SET_ONBOARDING_USER',
       username,
     );
-
+    isLogV2AppEvents() && dispatch(logEventAction('v2_ens_claimed'));
```

Comment:
I'm not sure that this is the right place - I don't actually know if we receive an event for this specifically, @poocart do you know? If we don't get an event we may need to rename this to something more relevant...